plugins/newsdownloader.koplugin/feed_config.lua

```diff
@@ -0,0 +1,8 @@
+return {
+    -- list your feeds here:
+    -- only supports http URL for now
+    -- Atom is currently not supported, only RSS
+    -- set limit to "0" means no download, "-1" no limit.
+    { "http://www.huffingtonpost.com/feeds/index.xml", limit = 2 },
```

Comment:
Quick reserch and Associated Press look's best for me:
AP - http://hosted.ap.org/lineups/WORLDHEADS.rss?SITE=AP&SECTION=HOME

Other proposal:
- ~Reuters http://feeds.reuters.com/Reuters/worldNews?format=xml~ - use atom
- ~CNN http://rss.cnn.com/rss/edition_world.rss~ use atom, not sure if neutral
- ~NYT http://rss.nytimes.com/services/xml/rss/nyt/World.xml~ as above
- AP sport news http://hosted2.ap.org/atom/APDEFAULT/347875155d53465d95cec892aeb06419
- economist international news http://www.economist.com/sections/international/rss.xml - TODO check if neutral,
http://www.news.com.au/sport/rss